### DISCUSSION TITLE
K40 gamer edition Skyline

### DISCUSSION THREAD

**Discussion by Robert281250** [skyline-emu/skyline#1186] [Game Requests]
Tengo el k40 gamer edition y no me abre el cuphead se queda en negro la pantalla no me deja jugar ningún juego alguien sabe el por qué ?

**Comment by AguileraJosue** [comment]
Hola, el k40 gaming usa una GPU Mali el cual muchos juegos no son compatibles aún. Deberás esperar a que el equipo optimice el emulador para GPUS Mali. Para la emulación te recomiendo un teléfono con GPU Adreno, Saludos✌️

**Reply by Robert281250** [reply]
El f3 es bueno ?es adreno

**Reply by AguileraJosue** [reply]
El Poco F3 deberia funcionarte bien🫶. Igualmente recuerda que el cuphead no esta optimizado del todo asi que no conseguirás moverlo a velocidad completa, pero si tendrás suficientes fps para que el juego sea bastante jugable sin problemas👐

**Reply by Robert281250** [reply]
Es que me. El k40 se me queda en negro la pantalla no lo corre para nada

**Reply by AguileraJosue** [reply]
Eso es por el tema que te explique del gpu Mali, en una gpu Adreno te funcionará 👐
![Screenshot_2022-05-06-00-54-21-246_skyline emu](https://user-images.githubusercontent.com/77398877/167069483-5e9cb2a8-6408-4ccd-a24f-956c89b7cf7f.jpg)
Aqui lo estoy emulando con un snapdragon 778G adreno!

**Comment by PixelyIon** [comment]
This is an English-only forum and device-specific bugs are not accepted, issues with Mali are known already and are being fixed in #987.

**Comment by IAn127388** [comment]
Hola dónde puedo descargar los juegos?
